src/atoms/Calendar/Dropdown.tsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
Can you disable this rule globally in .eslintrc.js with 'react/jsx-props-no-spreading': 'off'
You shouldn't do this normally but I think prop spreading is fine when used appropriately.